tests/responses/test_fileresponse.py

```diff
@@ -14,12 +14,14 @@ def test_file_from_disk_response(self):
         response = FileResponse(open(__file__, 'rb'))
         self.assertEqual(response['Content-Length'], str(os.path.getsize(__file__)))
         self.assertIn(response['Content-Type'], ['text/x-python', 'text/plain'])
+        self.assertEqual(response['Content-Disposition'], 'inline; filename="test_fileresponse.py"')
         response.close()
 
     def test_file_from_buffer_response(self):
         response = FileResponse(io.BytesIO(b'binary content'))
         self.assertEqual(response['Content-Length'], '14')
         self.assertEqual(response['Content-Type'], 'application/octet-stream')
+        self.assertEqual(response['Content-Disposition'], 'inline')
```

Comment:
Is this the behaviour we want when there's no filename? 

[From the RFC](https://tools.ietf.org/html/rfc6266#section-4.2):

> ...if it matches "inline" (case-insensitively), this
   implies default processing.  Therefore, the disposition type "inline"
   is only useful when it is augmented with additional parameters, such
   as the filename...

i.e. if we're not augmenting it, it's not useful. 

[The examples on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#Syntax) don't show the `inline; filename` syntax as an option. Happy that it is valid, but do we want to **always** include it?

Reply:
I agree that it's unnecessary.
I think I did it this way to make the code nice and clean, because the alternative I was thinking about wouldn't be as clean, but now I realized there's a clean enough way to do it without setting it in each case, so I changed it appropriately.